src/platforms/php/guides/symfony/index.mdx

```diff
@@ -61,12 +61,24 @@ return [
 
 ### Monolog Integration
 
-If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the normal error listener approach, you also need these additional configuration and services to log the errors correctly:
+If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the normal error listener approach, you also need this additional configuration to log the errors correctly:
```

Comment:
```suggestion
If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the typical error listener approach, you need this additional configuration to log the errors correctly:
```